src/panels/ActionsPanel/ActionsPanel.tsx

```diff
@@ -119,9 +128,21 @@ export default function ActionsPanel(): JSX.Element {
             setActionData(actions.results[0].actions);
           }
           setFetchingActionData(false);
+          setInlineErrors((prevInlineErrors) => {
+            const newInlineErrors = [...prevInlineErrors];
+            newInlineErrors[0] = null;
+            return newInlineErrors;
+          });
           return;
         })
-        .catch((error) => console.error(Label.GET_ACTIONS_ERROR, error));
+        .catch((error) => {
```

Comment:
I think this also needs to call `setFetchingActionData(false);` so that the spinner hides when the error appears.